app/app/api/private-cloud/products/[licencePlate]/count/_operations/count.ts

```diff
@@ -0,0 +1,25 @@
+import prisma from '@/core/prisma';
+
+export async function getCommentCountOp(licencePlate: string, requestId?: string) {
+  if (requestId) {
+    const commentCount = await prisma.privateCloudComment.count({
+      where: { requestId },
```

Comment:
I know it might not be necessary, but let's add `projectId` to the filter to ensure its uniqueness.

Reply:
Only problem is that adding `projectId` to the filter will not work for `create requests` as they do not have a `projectId` until that create request has been approved...